jupyter-docker/spark-defaults-temp.conf

```diff
@@ -0,0 +1,28 @@
+#######################
```

Comment:
I think `spark-defaults.conf.template` would be a clearer description of how this file works